pkgs/build-support/appimage/default.nix

```diff
@@ -205,6 +205,7 @@ rec {
       at-spi2-core
       pciutils # for FreeCAD
       pipewire # immersed-vr wayland support
+      xdg-user-dirs # flutter desktop apps
```

Comment:
If this is a command line tool, it shouldn't be grouped with "libraries not on the upstream include list".

Reply:
I don't know, but I'm sure somebody who knows more about appimage than me will — I just noticed the comment and thought it looked like it was in the wrong place :)